pkg/middlewares/aws_bucket_policy_expander.go

```diff
@@ -27,11 +27,14 @@ func (m AwsBucketPolicyExpander) Execute(_, resourcesFromState *[]resource.Resou
 			continue
 		}
 
-		bucket, _ := res.(*aws.AwsS3Bucket)
+		bucket, ok := res.(*resource.AbstractResource)
+		if !ok {
+			return nil
+		}
```

Comment:
Not sure we should return `nil` here. What could be the reason here to not have an `AbstractResource` ?